### DISCUSSION TITLE
screen problem

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3149] [Q&A] [answered]
I don't know when but ı open my computer and see this problem ı reset qtile but every time computer open and this problem comes
is this bug ? or my config file problem ?
https://github.com/ArchKubi/MyQtile/blob/main/low_resolution/qtile/config.py

![2021-12-29-064603_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/147626810-90e9432f-d645-4594-9bd1-978fed954519.png)
![2021-12-29-064605_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/147626813-e5789c17-3643-43b7-b075-170d0b8a967a.png)

**Comment by elParaguayo** [comment, marked as answer]
That does look a bit odd. 

Have you recently upgraded qtile to 0.19?

Are you setting screen resolution via `xrandr` in your startup script?

Does that issue disappear when you restart Qtile?

EDIT: I can see you are setting the screen with `xrandr` in your startup scripts so I've moved this to an issue (#3150) as I think it's the same as another one that we've seen.

**Reply by ghost** [reply]
yes like qtile 0.19.0 and xrandr issue
